api/task.go

```diff
@@ -487,6 +489,35 @@ func (h TaskHandler) Cancel(ctx *gin.Context) {
 	h.Status(ctx, http.StatusAccepted)
 }
 
+// CancelList godoc
+// @summary Cancel multiple tasks.
+// @description Cancel multiple tasks by IDs.
+// @tags tasks
+// @success 202
+// @router /tasks/cancel/list [put]
```

Comment:
Semantically, I think it would be more correct for the PUT to be against the collection because the action is updating the collection.  Ex: PUT /tasks/cancel (without the list).  The collection implies multiple objects.